public/assets/sass/pages/resources/exam-bank.scss

```diff
@@ -0,0 +1,121 @@
+@import "../../mixins/_banner-image.scss";
+
+#content {
+  max-width: var(--content-width);
+  margin: auto;
+  padding: 0 2em;
+}
+
+#exam-bank-searchbar {
+  margin: auto;
+  text-align: center;
+  padding: 1em;
+
+  #exams-search-label {
+    display: block;
+    font-size: 1.2em;
+    font-weight: bold;
+    margin: 0.5em 0 1em;
+  }
+
+  input {
```

Comment:
any change you could make the placeholder a bit lighter? the "CS 136" placeholder seems like its actually written in the input box rather than looking like a placeholder